### ISSUE TITLE
Webmaker communication for events 

### ISSUE THREAD

**Issue by LauraReynal** [MozillaFoundation/research#91]
Task: decide what is to be done for supporting Webmaker events. Write up a communication and share with community members. 
Team: @secretrobotron @HPaulJohnson @KevZawacki

**Comment by LauraReynal** [comment]
Meeting request sent, let's make decision and plan on monday.

**Comment by LauraReynal** [comment]
@HPaulJohnson @KevZawacki Thanks for writing the letter ! 
Made some suggestions in the doc.
Other than that, it's good to go for me :pineapple:

**Comment by KevZawacki** [comment]
Feedback incorporated; latest draft [here](https://docs.google.com/document/d/1uUYf4L_UPWSxpysdsXw6gswDYogBqidLMt74p4_pNt0/edit).

@amirad, @davidascher, @chrislarry33   Can you take a quick look and offer feedback? 

**TL;DR** We continue to receive funding requests for Webmaker-specific events. We need a response as to why we will no longer continue to support them. This language won't live on a blog or be leveraged as an official, public declaration -- it's just a note to select community members.

**Comment by davidascher** [comment]
how much money are we talking about?  In my opinion, a little bit of money
that keeps people engaged is money well spent, until we have other things
ready for them to do.

**Comment by HPaulJohnson** [comment]
Most requests are in the ~$200 range and there's a steady flow (mutliple per week across a bunch of different MoFo'ers). 

There's also a fair bit of overhead associated with actually getting payments processed as we never really perfected that system. 

All that said, I'm open to continuing to fund requests for the time being if that's preferable.

**Comment by secretrobotron** [comment]
:+1: love the draft, @KevZawacki, @HPaulJohnson. It sounds really encouraging.

@davidascher it does consume a lot of our (the people on this thread) time to process each one of these requests, and the __impact__ associated with these events is less clear than it was.

I think connecting directly to the reps/community/staff/others that are requesting event funding with the above letter will make everyone feel great about the initial launch, and give us time to position ourselves for something better as soon as we have the time.

TL;DR: communication good; continued, unfocused funding is confusing (for ...this project)

@amirad, @chrislarry33: would love your feedback here.

**Comment by davidascher** [comment]
I'm more than fine with not spending money on webmaker events. Do we have
emerging plans for ways to keep these community hubs engaged or is my
concern re dropping their engagement overblown?
